external/fv3kube/fv3kube/base_yamls/SHiELD/PIRE-like/fv3config.yml

```diff
@@ -0,0 +1,295 @@
+data_table: default
+diag_table: no_output
+experiment_name: default
+field_table: gs://vcm-fv3config/config/field_table/TKE-EDMF/v1.1/field_table
+forcing: gs://vcm-fv3config/data/base_forcing/SHiELD/v1.0/C48
+initial_conditions: ''  # no default provided
+namelist:
+  amip_interp_nml:
+    data_set: reynolds_oi
+    date_out_of_range: climo
+    interp_oi_sst: true
+    no_anom_sst: false
+    use_ncep_ice: false
+    use_ncep_sst: true
+  atmos_model_nml:
+    blocksize: 24
+    chksum_debug: false
+    dycore_only: false
+    fdiag: 3.0
+    first_time_step: false
+  coupler_nml:
+    atmos_nthreads: 1
+    calendar: julian
+    current_date:
+    - 2016
+    - 8
+    - 1
+    - 0
+    - 0
+    - 0
+    days: 1
+    dt_atmos: 900
+    dt_ocean: 900
+    hours: 0
+    minutes: 0
+    months: 0
+    seconds: 0
+    use_hyper_thread: true
+  diag_manager_nml:
+    prepend_date: false
+  external_ic_nml:
+    checker_tr: false
+    filtered_terrain: true
+    gfs_dwinds: true
+    levp: 64
+    nt_checker: 0
+  fms_io_nml:
+    checksum_required: false
+    max_files_r: 100
+    max_files_w: 100
+  fms_nml:
+    clock_grain: ROUTINE
+    domains_stack_size: 3000000
+    print_memory_usage: false
+  fv_core_nml:
+    a_imp: 1.0
+    adjust_dry_mass: false
+    beta: 0.0
+    consv_am: false
+    consv_te: 0.0
+    d2_bg: 0.0
+    d2_bg_k1: 0.2
+    d2_bg_k2: 0.1
+    d4_bg: 0.15
+    d_con: 1.0
+    d_ext: 0.0
+    dddmp: 0.5
+    delt_max: 0.002
+    dnats: 1
+    do_vort_damp: true
+    dwind_2d: false
+    ecmwf_ic: false
+    external_ic: false
+    fill: true
+    fill_dp: true
+    fv_debug: false
+    fv_sg_adj: 600
+    gfs_phil: false
+    grid_type: 0
+    hord_dp: -5
+    hord_mt: 5
+    hord_tm: 5
+    hord_tr: -5
+    hord_vt: 5
+    hydrostatic: false
+    io_layout:
+    - 1
+    - 1
+    k_split: 1
+    ke_bg: 0.0
+    kord_mt: 9
+    kord_tm: -9
+    kord_tr: 9
+    kord_wz: -9
```

Comment:
@lharris4 notes that following https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/302 using `kord_wz < 0` can occasionally lead to instability.  He recommends potentially switching to `kord_wz = 9` instead, which we could consider doing preemptively (instability or not, https://github.com/NOAA-GFDL/GFDL_atmos_cubed_sphere/pull/302 already changes answers with `kord_wz = -9` relative to the PIRE simulation codebase).  

> A couple of months ago there was an update to the kord_wz < 0 code to fix a memory bug (I think you found this, thank you), but since then I have been having stability problems with this code which is odd. I think you might be able to try kord_wz < 0 but if it causes stability problems in the most recent codes you should use kord_wz > 0. Ultimately there is very little scientific difference between the two, it is mostly that kord_wz < 0 more closely matches the lower boundary condition used in the semi-implicit solver.